LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Mattias Erming
```

Comment:
> @JocelynDelalande, what I am sure about though, is that I want this PR to make it into master quickly :-) Can I get your +1 and continue the discussion to reach to a solution even if it has to be done post-merged, in another PR?

Sure :) so please reflect current project state : copyright holder is "Mathias Erming and contributors", I was not discussing if we wanted a CLA or not, that's outside the scope of this PR.